packages/runtime/src/next/components/tests/controls/page-control-prop-rendering.tsx

```diff
@@ -9,13 +9,9 @@ import { type Data, type ValueType, type DataType, ControlDefinition } from '@ma
 
 import { ElementData } from '../../../../state/react-page'
 import { Page } from '../../page'
-import { ReactRuntimeProvider } from '../../../../runtimes/react'
 import { ReactRuntime } from '../../../../react'
 
-import {
-  createMakeswiftPageSnapshot,
-  createRootComponent,
-} from '../../../../utils/tests/element-data-test-test'
+import * as Testing from '../../../../runtimes/react/testing'
```

Comment:
Here and below, update rendering tests to use the [newly minted `<Testing.ReactProvider>`](https://github.com/makeswift/makeswift/pull/927/files#r1932966780).